docs/examples/battle.jl

```diff
@@ -182,7 +182,7 @@ function captor_behavior!(agent, model)
         ## Prisoner runs away in the commotion
         prisoner.shape = :utriangle
         prisoner.capture_time = 0
-        walk!(prisoner, (rand(abmrng(model), -1:1, 2)..., 0), model)
+        walk!(prisoner, (rand(abmrng(model), -1:1, 2)..., 0), model, ifempty=false)
```

Comment:
```suggestion
        walk!(prisoner, (rand(abmrng(model), -1:1, 2)..., 0), model; ifempty=false)
```

i believe it makes for a more readable code if keywords are separated by `;` can you change this here and in subsequent calls?

Reply:
Added this change in new commit